modules/rosa-sts-aws-requirements.adoc

```diff
@@ -0,0 +1,37 @@
+[id="rosa-sts-customer-requirements_{context}"]
+= Customer requirements when using STS for deployment
+
+The following prerequisites must be complete before you deploy {product-title} (ROSA) clusters using STS.
+
+[id="rosa-account_{context}"]
+== Account
+* The customer ensures that the link:https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html[AWS limits] are sufficient to support {product-title} provisioned within the customer's AWS account.
+* If SCP policies are applied and enforced, these policies must not be more restrictive than the roles and policies required by the cluster.
+* The customer's AWS account should not be transferable to Red Hat.
+* The customer should not impose additional AWS usage restrictions beyond the defined roles and policies on Red Hat activities. Imposing restrictions will severely hinder Red Hat's ability to respond to incidents.
+* The customer may deploy native AWS services within the same AWS account.
++
+[NOTE]
+====
+Customers are encouraged, but not mandated, to deploy resources in a Virtual Private Cloud (VPC) separate from the VPC hosting {product-title} and other Red Hat supported services.
+====
+
+[id="rosa-access-requirements_{context}"]
+== Access requirements
+
+* Red Hat must have AWS console access to the customer-provided AWS account. This access is protected and managed by Red Hat.
```

Comment:
Should we clarify the role arn that user need to add to their roles as trusted relationship in Access requirement?

Reply:
@xueli181114 Can you provide a little more clarification? I'm not quite sure what the bullet should say.  Is it " The user needs to add the  <arnrole  name>  role as a trusted relationship in  ___.

Reply:
I will close the comments. The detailed steps had been covered in another doc modules/rosa-sts-creating-roles.adoc